webpack/move_to_pf/test-utils/testHelpers.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
```

Comment:
I couldn't consume this file from core so I copied it here.
Should we implement an opinionated sub-package into `patternfly-react` that deliver those test utils?
The [IntegrationTestHelper.js](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/IntegrationTestHelper.js) may also belong there.

This sub-package goal may to deliver **tutorials, documentation and utils** for writing tests in our infrastructure.

Reply:
I would be okay with that.  I think it would be worth proposing to the patternfly-react community.